.husky/pre-commit

```diff
@@ -1 +1,4 @@
+#!/usr/bin/env.sh
+. "$(dirname -- "$0")/_/husky.sh"
 
+npx lint-staged
```

Comment:
Prefer to use npm vs npx.

Figure out why we should do that and explain first

Reply:
Changed to npm.

NPM is a package manager, hence it will locally install the packages before running them so their versions can be managed on a local level.

NPX is a package executor, hence it directly fetches the latest version of the package from the npm registry and executes them. It is good when we want to execute something just once.

In my case, it is better to install lint-staged locally so the version will stay consistent and any bugs from the latest versions will be avoided as well.

Thanks for pointing it out.